pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -167,3 +170,43 @@ func TestBinderApplicationName(t *testing.T) {
 	})
 
 }
+
+func TestKnativeServicesContractWithBinder(t *testing.T) {
+	ns := "binder"
```

Comment:
Could you make sure formatting is set to 4 spaces and following `.editorconfig`?

Reply:
In my editor, it is 4 spaces only. 
![spaces](https://user-images.githubusercontent.com/26519653/71371256-91595000-25d6-11ea-9870-8bba3b277876.png)

Reply:
At GitHub also it is 4 spaces, I think in reviewing its showing 2
![space](https://user-images.githubusercontent.com/26519653/71371386-f8770480-25d6-11ea-88fa-37db9ac13bf9.png)

Reply:
Try to view it as a file.